AI Demos/Summarizer/Summarizer/AssistServices.cs

```diff
@@ -0,0 +1,162 @@
+﻿using Syncfusion.Maui.AIAssistView;
+using System.Collections.ObjectModel;
+using Azure.AI.OpenAI;
+using Azure;
+
+namespace Summarizer
+{
+    /// <summary>
+    /// Represents a class providing AI assistance services for text extraction, 
+    /// prompt generation, and chat completion functionalities using OpenAI's GPT model.
+    /// </summary>
+    internal class AssistServices
+    {
+
+        /// <summary>
+        /// Gets or sets the extracted text from a document.
+        /// </summary>
+        internal string? ExtractedText { get; set; }
+
+        /// <summary>
+        /// The EndPoint
+        /// </summary>
+        private string endpoint = "https://yourendpoint.com/";
+
+        /// <summary>
+        /// The Deployment name
+        /// </summary>
+        internal string deploymentName = "DEPLOYMENT_NAME";
```

Comment:
The casing should be proper for internal variable.

Reply:
Updated bro